Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciooUno
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            // se instancia pedro             
+            Pedro opedro = new Pedro ();
+
+            // se crea modelo persona de pedro
+            ModeloPersona modelopedro = new ModeloPersona ("Blanco",45,"pedro perez");
+
+            //se llena info de pedro
+            opedro.llenarInfo (modelopedro);
+
+            // Se llama tocar piano           
+            opedro.tocarPiano();
+
+            //se llama caminar
+            opedro.caminar();
+
+            // se muestra informacion de pedro almacenada
+
+            Console.Write("pedro tiene el pelo de color " + opedro.variableGlobal.ColorPelo);
```

Comment:
You can wrap these three console into one by doing


 Console.Write("pedro tiene el pelo de color " + opedro.variableGlobal.ColorPelo + " su edad es "+ opedro.variableGlobal.Edad +" y su nombre es " + opedro.variableGlobal.Nombre );

Reply:
has been corrected